### ISSUE TITLE
Fix S2696 FN: Support unsigned right-shift operator (>>>)

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6304]
### Description

FN for S2696 (StaticFieldWrittenFromInstanceMember).

Using C#11 unsigned right-shift operator (>>>), the rule is not raising an issue.

### Repro steps

```csharp
    class StaticFieldWrittenFromInstanceMember
    {
        private static int count = 0;
        public int MyProperty
        {
            get { return myVar; }
            set
            {
                count >>>= 42; // FN
                myVar = value;
            }
        }
    }
```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
